tensorflow/core/kernels/map_kernels.h

```diff
@@ -175,6 +176,31 @@ class TensorMapHasKey : public OpKernel {
   }
 };
 
+template <typename Device>
+Status TensorMapBinaryAdd(OpKernelContext* c, const TensorMap& a,
+                          const TensorMap& b, TensorMap* out) {
+  // binary add returns a union of keys, values with keys in the intersection are added
```

Comment:
Style nit (for future changes; to expedite the submission, there's no need to update this one, unless other changes will be needed) - we prefer code comments to be well-formed sentences, with capitalization. Punctuation can be omitted if the comment is very short (2-3 words).